asapdiscovery-docking/asapdiscovery/docking/cli.py

```diff
@@ -131,5 +135,81 @@ def large_scale(
     large_scale_docking_workflow(inputs)
 
 
+@docking.command()
+@target
+@click.option(
+    "--use-omega",
+    is_flag=True,
+    default=False,
+    help="Whether to use OEOmega conformer enumeration before docking (slower, more accurate)",
+)
+@click.option(
+    "--allow-retries",
+    is_flag=True,
+    default=False,
+    help="Whether to allow POSIT to retry with relaxed parameters if docking fails (slower, more likely to succeed)",
+)
+@click.option(
+    "--allow-final-clash",
+    is_flag=True,
+    default=False,
+    help="Allow clashing poses in last stage of docking",
+)
+@ligands
+@pdb_file
+@fragalysis_dir
+@structure_dir
+@gen_cache
+@cache_dir
+@cache_type
+@dask_args
+@output_dir
+@input_json
+def cross_docking(
+    target: TargetTags,
+    use_omega: bool = False,
```

Comment:
Make default here same as in base class as will be overridden. 

We need to be careful about being overidden by CLI args also.

Reply:
bah yeah this is a bit of a concern. should be ok though bc the overriding goes in the direction that makes sense (cli --> workflow --> class).

Reply:
Perhaps @jthorton knows a design that circumvents this?